src/components/PostList/PostList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './PostList.css';
+
+import PropTypes from 'prop-types';
+
+import { Post } from '../Post/Post';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(current => (
+      <Post
+        key={current.post.id}
+        data={current}
```

Comment:
```suggestion
        post={current}
```
data is too generic, use `post` as props name, and consider using spread operator to pass it